cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -743,9 +771,25 @@ fn aarch64_get_operands<F: Fn(VReg) -> VReg>(inst: &Inst, collector: &mut Operan
         }
         &Inst::FpuRRI { fpu_op, rd, rn, .. } => {
             match fpu_op {
-                FPUOpRI::UShr32(..) | FPUOpRI::UShr64(..) => collector.reg_def(rd),
-                FPUOpRI::Sli32(..) | FPUOpRI::Sli64(..) => collector.reg_mod(rd),
+                FPUOpRI::UShr32(..) | FPUOpRI::UShr64(..) => {}
+                FPUOpRI::Sli32(..) | FPUOpRI::Sli64(..) => {
```

Comment:
Similarly here - how about introducing a separate enum, say `FpuRRIModOp`, for these operations?